patches/components-permissions-permission_uma_util.cc.patch

```diff
@@ -10,3 +10,11 @@ index 028a47c6f23892e7fc1e33f742717d379543c3be..3e256e549eb7747113fb8c76cfdbc3cf
    }
  }
  
+@@ -175,6 +176,7 @@ RequestTypeForUma GetUmaValueForRequests(
```

Comment:
@goodov do we actually need this stuff for anything? Why don't we just override PermissionUmaUtil to be a no-op for everything?